.circleci/config.yml

```diff
@@ -70,8 +78,7 @@ jobs:
       - run: make muzzle
 
   release-bintray-publish:
-    docker:
-      - image: cimg/openjdk:11.0
+    executor: gradle_docker
```

Comment:
This should fix issue

```
#!/bin/bash -eo pipefail
echo $DOCKERHUB_PASSWORD | docker login --username $DOCKERHUB_USERNAME --password-stdin
Error: Cannot perform an interactive login from a non TTY device

Exited with code exit status 1
CircleCI received exit code 1
```

from https://app.circleci.com/pipelines/github/hypertrace/javaagent/466/workflows/5f098bef-482c-4144-85fa-eb45d3ad953c/jobs/1020